### PR TITLE
Disable menu items on empty screen / data (EXPOSUREAPP-13615)

### PR DESCRIPTION
**Pull request by schauersbergern**
[Ticket](https://jira-ibs.wbs.net.sap/browse/EXPOSUREAPP-13615)

### CODE REVIEW

**Comment by SamuraiKek** [comment]
Works well but I think you missed the last bulletpoint in the ticket description.

> the edit screen of contact persons and locations is dismissed automatically after the last item is deleted

The edit screen stays on after deleting all people/places.

File: Corona-Warn-App/src/main/java/de/rki/coronawarnapp/contactdiary/ui/overview/ContactDiaryOverviewViewModel.kt [hunk 1]
```diff
@@ -94,6 +94,9 @@ class ContactDiaryOverviewViewModel @AssistedInject constructor(
     private val traceLocationCheckInRiskFlow = riskLevelStorage.traceLocationCheckInRiskStates
     private val checkInsWithinRetentionFlow = checkInRepository.checkInsWithinRetention
 
+    val locations = contactDiaryRepository.locations.asLiveData(dispatcherProvider.Default)
+    val people = contactDiaryRepository.people.asLiveData(dispatcherProvider.Default)
```
Comment (mtwalli) [inline Corona-Warn-App/src/main/java/de/rki/coronawarnapp/contactdiary/ui/overview/ContactDiaryOverviewViewModel.kt:98]: `asLiveData2` ? [optional]

**Comment by schauersbergern** [comment]
> Works well but I think you missed the last bulletpoint in the ticket description.
> 
> > the edit screen of contact persons and locations is dismissed automatically after the last item is deleted
> 
> The edit screen stays on after deleting all people/places.

@SamuraiKek thanks for the hint, last bulletpoint is implemented now

File: Corona-Warn-App/src/main/java/de/rki/coronawarnapp/contactdiary/ui/overview/ContactDiaryOverviewViewModel.kt (hunk 1 above)
Comment (schauersbergern) [reply Corona-Warn-App/src/main/java/de/rki/coronawarnapp/contactdiary/ui/overview/ContactDiaryOverviewViewModel.kt]: fixed

**Review by mtwalli — APPROVED** [review]

**Pull request merged**